schemas/questions/types/general.json

```diff
@@ -35,9 +35,75 @@
         "type": "string",
         "enum": ["General"]
       },
+      "dynamic_answers": {
+        "type": "object",
+        "properties": {
+          "values": {
+            "type": "object",
+            "source": ["list"],
+            "identifier": {
+              "$ref": "../../common_definitions.json#/non_empty_string"
+            }
+          }
+        },
+        "answers": {
+          "type": "array",
+          "items": {
+            "oneOf": [
+              {
+                "$ref": "../../answers/address.json#/answer"
+              },
+              {
+                "$ref": "../../answers/checkbox.json#/answer"
+              },
+              {
+                "$ref": "../../answers/currency.json#/answer"
+              },
+              {
+                "$ref": "../../answers/date.json#/answer"
+              },
+              {
+                "$ref": "../../answers/month_year_date.json#/answer"
+              },
+              {
+                "$ref": "../../answers/year_date.json#/answer"
+              },
+              {
+                "$ref": "../../answers/duration.json#/answer"
+              },
+              {
+                "$ref": "../../answers/number.json#/answer"
+              },
+              {
+                "$ref": "../../answers/percentage.json#/answer"
+              },
+              {
+                "$ref": "../../answers/radio.json#/answer"
+              },
+              {
+                "$ref": "../../answers/text_area.json#/answer"
+              },
+              {
+                "$ref": "../../answers/text_field.json#/answer"
+              },
+              {
+                "$ref": "../../answers/unit.json#/answer"
+              },
+              {
+                "$ref": "../../answers/dropdown.json#/answer"
+              },
+              {
+                "$ref": "../../answers/relationship.json#/answer"
+              },
+              {
+                "$ref": "../../answers/mobile_number.json#/answer"
+              }
+            ]
+          }
+        }
+      },
       "answers": {
         "type": "array",
-        "minItems": 1,
```

Comment:
This shouldn't have changed. If we have answers, then it should be min 1.

You need to update the JSON validation to be conditional answer if dynamic answers is defined. One of them must be defined.

Reply:
Still outstanding.

Mine items can stay where it is.

You need to make add JSON logic to check at least either dynamic_answers or answers are defined.

Reply:
Added this logic now, added back min items.